flit.ini

```diff
@@ -4,5 +4,11 @@ author = Thomas Kluyver
 author-email = thomas@kluyver.me.uk
 home-page = https://github.com/takluyver/entrypoints
 classifiers = License :: OSI Approved :: MIT License
+    Programming Language :: Python :: 2
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3
```

Comment:
I prefer to just use the `2` and `3` classifiers, leaving out the more specific versions. They only get outdated when new versions of Python come out, and we have the `Requires-Python` metadata now for the same purpose.

Reply:
Sure, I've removed those.

Should Requires-Python be more specific to prevent pip attempting to install it on the low 3.x versions?

Reply:
I wouldn't bother. It probably still works on Python 3.3 for now, so there's no need to prevent people from installing it, and basically no-one is using anything older than that. I'll merge this now, thanks!